configs/sites/frontera/packages.yaml

```diff
@@ -0,0 +1,256 @@
+packages:
+  all:
+    compiler:: [intel@23.1.0]
+    #compiler:: [gcc@12.2.0]
```

Comment:
Use either the gcc version you configured in `compilers.yaml`, or delete the line.

Reply:
Okay. I removed that part. Once we have working Intel installation, I might start working on bringing GNU support too.